modules/BooksFormat.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-restricted-globals */
+export default class BookList {
+  constructor(bookDisplayContainer) {
```

Comment:
- [OPTIONAL] I just suggest removing `/* eslint-disable no-restricted-globals */` because It's not a good practice to disable the linter using inline-comments.

> To Fix this linter error, You've to use `window.location.reload()` instead of `location.reload()` in line 37 💡

```js
window.location.reload();
```